gammapy/modeling/fit.py

```diff
@@ -521,10 +518,11 @@ class CovarianceResult(FitResult):
 class OptimizeResult(FitResult):
     """Optimize result object."""
 
-    def __init__(self, nfev, total_stat, trace, **kwargs):
+    def __init__(self, nfev, total_stat, trace, covariance_results=None, **kwargs):
         self._nfev = nfev
         self._total_stat = total_stat
         self._trace = trace
+        self.covariance_results = covariance_results
```

Comment:
Just rename to `covariance_result`? I think the plural is misleading here, because there is only a single result...